lib/spack/spack/build_systems/python.py

```diff
@@ -319,6 +320,9 @@ def test(self):
         # Make sure we are importing the installed modules,
         # not the ones in the source directory
         for module in self.import_modules:
+            if not re.match('[a-zA-Z0-9._]+$', module):
```

Comment:
Can we instead move this logic into `self.import_modules`?

Reply:
Sure.